src/services/indoor-path-planning.service.ts

```diff
@@ -156,19 +156,19 @@ class IndoorPathPlanningService {
    * @param object any time that has an id
    */
   private getPOI = (object: MarkerLocation) => {
```

Comment:
Taking my approval back. Please change this method name to `getMarker` since it no longer returns a simple `POI`